test/ipfs_mock.go

```diff
@@ -317,6 +322,29 @@ func (m *IpfsMock) handler(w http.ResponseWriter, r *http.Request) {
 			goto ERROR
 		}
 		w.Write(data)
+	case "repo/gc":
```

Comment:
add comment saying it assumes stream-errors=true. Actually, you may want to set one of the errors.